exercises/bankaccount.js

```diff
@@ -3,21 +3,35 @@
 class Account{
 	constructor(){
 		//store the amount of money in the account
+		this.balance = 0;
 	}
-	add(  ){
+	add( amount ){
 		//add money to the amount stored in the account
+		this.balance += amount;
+		return this.balance;
 		//takes in an amount
 		//adds it to the existing amount
 		//returns the new amount in the account
 	}
 	remove( amount ){
-		//removes funds from an account
+		//removes funds from an account	
+		var currentBalance = this.balance;
+		if(amount > this.balance){
+			this.balance = 0;
```

Comment:
in both cases you re changing the balance.  It might be better to define how much you're going to take out of the balance in the if statement, then deduct from the balance just once